findingaids/templates/findingaids/finding_aids_page.html

```diff
@@ -3,189 +3,266 @@
 
 {% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}
 {% block content %}
-  <article>
+    <article>
 
-    <div class="row">
-      <div class="col-xs-12">
-        <div class="btn-group spaces-toggle">
-          <a class="btn btn-list-toggle{% if view == 'title' %} active{% endif %}" href="?view=title">By Title</a>
-          <a class="btn btn-list-toggle{% if view == 'topics' %} active{% endif %}" href="?view=topics">By Topic</a>
-          <a class="btn btn-list-toggle{% if view == 'digitized' %} active{% endif %}" href="?view=digitized">With Digital Content</a>
-        </div>
-      </div>
+        <div class="row">
+            <div class="col-xs-12">
+                <div class="btn-group spaces-toggle">
+                    <a class="btn btn-list-toggle{% if view == 'title' %} active{% endif %}" href="?view=title&browse=all">By Title</a>
+                    <a class="btn btn-list-toggle{% if view == 'topics' %} active{% endif %}" href="?view=topics">By Topic</a>
+                    <a class="btn btn-list-toggle{% if view == 'digitized' %} active{% endif %}" href="?view=digitized">With Digital Content</a>
+                </div>
+            </div>
 
-      <div class="col-xs-12 col-sm-6 col-sm-offset-3 distinct-search">
-        <form action="." class="searchbox" method="get">
-          <input type="search" required="" onkeyup="buttonUp();" class="searchbox-input" name="searchq" placeholder="Search finding aids"{% if searchq %} value="{{ searchq }}"{% endif %}/>
-          <span class="searchbox-icon"><i title="search" class="fa fa-search"></i>
-            <input type="submit" class="searchbox-submit" style="background-color: transparent; color: transparent;"/>
-          </span>
-        </form>
-     
-        <!-- 
-          <span class="headline">Limit to</span> &nbsp;
-          <input type="checkbox" value="ebooks" id="checkboxebooks"> Digitized Content
-        -->
-      </div>
+            <div class="col-xs-12 col-sm-6 col-sm-offset-3 distinct-search">
+                <form action="." class="searchbox" method="get">
+                    <input type="search" required="" onkeyup="buttonUp();" class="searchbox-input" name="searchq" placeholder="Search finding aids"{% if searchq %} value="{{ searchq }}"{% endif %}/>
+                    <span class="searchbox-icon"><i title="search" class="fa fa-search"></i>
+                        <input type="submit" class="searchbox-submit" style="background-color: transparent; color: transparent;"/>
+                    </span>
+                </form>
+            </div>
+            <div class="col-md-12">
+                <!-- just padding -->
+                <p/>
```

Comment:
I think this is technically syntactically valid but probably not good for accessibility. I think it will be interpreted as an open p: `<p>`, which is technically fine but not ideal. Either a SASS fix or `<br/>` would be better.

Reply:
Whoops; thanks for catching that; I intended to replace that with something more respectable and then forgot.